doc/UserGuide.md

```diff
@@ -53,6 +53,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorting all persons by name : `sort`
```

Comment:
It might be useful to tell the user how the list will be sorted (i.e. alphabetically by first name).